glass/src/libnt/native/include/glass/networktables/NetworkTables.h

```diff
@@ -95,8 +95,10 @@ enum NetworkTablesFlags_ {
   NetworkTablesFlags_ShowFlags = 1 << 3,
   NetworkTablesFlags_ShowTimestamp = 1 << 4,
   NetworkTablesFlags_CreateNoncanonicalKeys = 1 << 5,
-  NetworkTablesFlags_Default = 1 & ~NetworkTablesFlags_ReadOnly &
-                               ~NetworkTablesFlags_CreateNoncanonicalKeys
+  NetworkTablesFlags_Precision = 0xff << 6,
+  NetworkTablesFlags_Default = (1 & ~NetworkTablesFlags_ReadOnly &
+                                ~NetworkTablesFlags_CreateNoncanonicalKeys) |
+                               (6 << 6),
```

Comment:
Since this is more complex than a boolean flag, the shift and mask should be made constants (e.g. rather than manually having "6" everywhere).